packages/node_modules/pouchdb-replicator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "pouchdb-replicator",
+  "version": "2.3.4",
```

Comment:
this doesn't need to include a version; that's automatically added at `release` time.

OTOH we have to make sure we can actually publish a new version that's the same verison as express-pouchdb/pouchdb-server. Since the latest pouchdb-replicator is 2.1.3 looks like we're good.